core/src/main/java/de/bwaldvogel/mongo/backend/InMemoryCursor.java

```diff
@@ -29,4 +29,9 @@ public boolean isEmpty() {
         return documents;
     }
 
+    @Override
+    public Cursor invalidate() {
```

Comment:
the method name is misleading in my opinion. The method doesn’t really invalidate (and kill?) the current cursor but instead removes an "invalidated copy" of it

Reply:
I’ve refactored it in 5a2db323df70153d51afe30cb9bd8b0074990c2b, 6e78d5cd79d407a083484b56d9b8bc54cd65e4c9 and c4240afdd54d918974ce80274d0b8d6800dd6a11

Reply:
thanks, looks better now